workflow/tests/hpxml_translator_test.rb

```diff
@@ -512,6 +512,9 @@ def _verify_outputs(runner, rundir, hpxml_path, results)
       if hpxml_path.include?('base-simcontrol-timestep-10-mins.xml') || hpxml_path.include?('ASHRAE_Standard_140')
         next if err_line.include? 'Temperature out of range [-100. to 200.] (PsyPsatFnTemp)'
       end
+      if hpxml_path.include?('base-schedules-stochastic.xml') || hpxml_path.include?('base-schedules-user-specified.xml')
+        next if err_line.include?('GetCurrentScheduleValue: Schedule=') && err_line.include?('is a Schedule:File')
```

Comment:
@shorowit I suppose we need to account for DST in the generated schedule csv? And also remove `RunPeriodControl:DaylightSavingTime` any time `OccupancySchedulesCSVPath` exists in the hpxml file?

Reply:
Follow up with energyplus team on whether we can keep our existing schedules (and have RunPeriodControl:DaylightSavingTime object applied to them) AND use Schedule:File with adjustments built in.

Reply:
Should we just create a single csv file for every schedule? Nothing prevents us from doing that, but it could become slower. Perhaps could go this route when we are using generated stochastic schedules. Preference would be to have non Schedule:File with DST applied and Schedule:File with manual DST adjustments.

Reply:
Where did we end up on this? Nothing has to change on our end and the E+ warning is acceptable?

Reply:
That's what it sounds like. EnergyPlus is applying the shifting on schedules referenced by `Schedule:File` objects, and we want that to happen because we want our "local time" schedules to sync with "standard time".

I think the caveat, though, is that we must always apply DST (otherwise the schedules wouldn't shift and would therefore be incorrect). Not sure how to enforce/handle this. Ideas? Update EPvalidator.xml to prevent `DaylightSaving/Enabled=false` when `OccupancySchedulesCSVPath` is present?

See https://github.com/NREL/OpenStudio-BuildStock/issues/469 and https://github.com/NREL/EnergyPlus/issues/8300 for  reference.

@shorowit @nmerket

Reply:
There are certain locations that don't use DST, so I'd think those locations should continue to not have DST set, whether using the Schedule:File objects or otherwise (weekday/weekend multipliers). I'm not seeing why DST must always apply?

Reply:
@shorowit Yes, I think, we should not set DST for those locations. The schedules generated by the ScheduleGenerator are based on local time, so, in order for it to be applied correctly,  the DST needs to be set if the location practices DST, and not set if they don't. 

Further details and thinking out loud:
The schedules are based on the ATUS data which uses local time(which can have DST). However, we currently do not have location dependency on the model, so, the schedules are based on the average pattern throughout the US.

To use an example, if the average pattern is that people wake up at 6:00 am local time, (which would be 5:00 AM standard time during DST active period), the Schedule:File will have 6:00 am wake up schedules. For locations with DST, the shifting in E+ will make it 5:00 am standard time wakeup, so it works. For locations without DST, the DST is not set in the model, and so there will be no shifting and E+ will leave it at 6:00 am standard time wakeup, so, it works in this case too, because, the people in those locations really do wake up at 6:00 am standard time (since no DST).

Reply:
@rajeee So what does this mean for us? That we should "pre-shift" non-DST locations in the opposite direction so that when EnergyPlus shifts it makes up for it?

Reply:
@joseph-robertson I don't think we need to pre-shift. We will not be setting DST on non-DST locations right?